packages/api/src/routes/medical/dtos/network-entry-dto.ts

```diff
@@ -1,14 +1,15 @@
 import { normalizeState } from "@metriport/shared";
 import { NetworkEntry } from "../../../domain/medical/network-entry";
-import { CQDirectoryEntry2 } from "../../../external/carequality/cq-directory";
+import { HieDirectoryEntry } from "../../../external/hie/domain/hie-directory-entry";
 
 export type NetworkEntryDTO = NetworkEntry;
 
-export function dtoFromHieDirectoryEntry(networkEntry: CQDirectoryEntry2): NetworkEntryDTO {
+export function dtoFromHieDirectoryEntry(networkEntry: HieDirectoryEntry): NetworkEntryDTO {
   return {
+    id: networkEntry.id,
     name: networkEntry.name,
-    oid: networkEntry.id,
-    zip: networkEntry.zip,
+    oid: networkEntry.oid,
+    zipCode: networkEntry.zipCode,
     state: networkEntry.state ? normalizeState(networkEntry.state) : undefined,
```

Comment:
here's the normalize state call that should be able to process puerto rico state codes